test/unit_tests/test_config.js

```diff
@@ -102,6 +102,11 @@ describe('tracer config tests', () => {
         expect(config.getConfig()).to.contain({ decodeHTTP: false });
     });
 
+    it('setConfig: set skipReturnValue to true', () => {
```

Comment:
is it possible to add a functional test of this parameter?
(i.e return value is not added when this is true)